README.md

```diff
@@ -16,7 +16,8 @@ The `difference` block shows the difference between the two timers, i.e. `Time D
 
 NOTE: It is recommended to use this in laptops only. This project is not made responsive so, could not expect to give a good experience while using in mobile phones. If you wish to make it responsive then do shoot me a PR.<br><br>
 ## Contributing
-* Clone the repo: `git clone https://github.com/Raghwendra-Dey/productivity_meter.git`
+* fork the repo:
+* Clone the repo from the repo which you just have forked  and the command  should look like "git clone https://github.com/your_user_name/productivity_meter.git"
```

Comment:
Put the command in backticks to highlight it. Also surround `your_user_name` with curly braces, and make spell it in capitals.